pwa-devdocs/_drafts/toasts/useToastContext/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: useToastContext
+---
+
+<!--
+The reference doc content is generated automatically from the source code.
+To update this section, update the doc blocks in the source code
+-->
+
+{% include auto-generated/peregrine/lib/Toasts/useToastContext.md %}
+
+## Examples
+
+### Using the Toast context logic
+
+Import the ToastContextProvider and wrap it around components that use Toast data and functionality.
+
+```jsx
+// MyComponent.js
+
+import {ToastContextProvider} from '@peregrine/lib/Toasts/useToastContext'
```

Comment:
This should be `from '@magento/peregrine'`

Reply:
fixed